public/app/features/dashboard/components/GenAI/GenAIPanelDescriptionButton.tsx

```diff
@@ -12,48 +12,48 @@ interface GenAIPanelDescriptionButtonProps {
   panel: PanelModel;
 }
 
+const PANEL_DESCRIPTION_CHAR_LIMIT = 200;
+
 const DESCRIPTION_GENERATION_STANDARD_PROMPT =
   'You are an expert in creating Grafana Panels.\n' +
   'You will be given the title and description of the dashboard the panel is in as well as the JSON for the panel.\n' +
   'Your goal is to write a descriptive and concise panel description.\n' +
   'The panel description is meant to explain the purpose of the panel, not just its attributes.\n' +
   'Do not refer to the panel; simply describe its purpose.\n' +
   'There should be no numbers in the description except for thresholds.\n' +
-  'The description should be, at most, 140 characters.';
+  `The description should be, at most, ${PANEL_DESCRIPTION_CHAR_LIMIT} characters.`;
 
 export const GenAIPanelDescriptionButton = ({ onGenerate, panel }: GenAIPanelDescriptionButtonProps) => {
-  const messages = React.useMemo(() => getMessages(panel), [panel]);
```

Comment:
Removing this memo fixes the stale state issue - not the most optimal solution as I believe this is also causing maximum call update issue potentially.. but does fix immediate problem for time being 🤷‍♂️